tremor-script/src/ast/support.rs

```diff
@@ -24,60 +21,13 @@ impl<'script> fmt::Debug for InvokeAggrFn<'script> {
     }
 }
 
+/// custom impl becauyse field `invocable` does not implement `PartialEq`
```

Comment:
could a blanket implementation of `impl<T: AstEq> for PartialEq T` work here?

Reply:
you mean that `PartialEq` used the `AstEq` logic for everything that also implements `AstEq`?

If so, I would say no, as `AstEq` explicitly ignores the node metadata (`mid`) and thus e.g. the location in the source, while `PartialEq` should take the `mid` into account.

Reply:
fair argument!